src/Table/TableWithSorting.story.tsx

```diff
@@ -1,92 +0,0 @@
-/* eslint-disable react/prop-types */
```

Comment:
Do you think it's worth keeping this example, just renaming it to "With Custom Sorting", to show how one would implement sorting that is not the standard in `<SortingTable />` if they needed to?

Reply:
I was actually wondering the same this morning. Originally we were just going to fix the example, but went the route of making the `SortingTable` component.
But I agree, it could still be useful to someone. I'll add it back in.

Reply:
Done